tests/conftest.py

```diff
@@ -0,0 +1,36 @@
+import django
+from stellar_sdk import Keypair
+
+
+def pytest_configure(config):
```

Comment:
Suggestion - `pytest_configuration` or, if it needs to be a verb, `set_up_pytest` (I think set up is more common than configure).

Reply:
In this case, this function is a hook from `pytest` to perform the initial configuration for third-party libraries.

https://docs.pytest.org/en/latest/how-to/writing_hook_functions.html#optionally-using-hooks-from-3rd-party-plugins
https://newbedev.com/how-to-run-a-method-before-all-tests-in-all-classes